GoogleDataTransport/GDTCORLibrary/GDTCORTransformer.m

```diff
@@ -73,35 +73,17 @@ - (void)transformEvent:(GDTCOREvent *)event
       }
     }
     [self.storageInstance storeEvent:transformedEvent];
-    if (self->_runningInBackground) {
-      [[GDTCORApplication sharedApplication] endBackgroundTask:bgID];
-      bgID = GDTCORBackgroundIdentifierInvalid;
-    }
+    [[GDTCORApplication sharedApplication] endBackgroundTask:bgID];
```

Comment:
I know this is different from what I had, but a check for validity is probably good.

`if (bgID != GDTCORBackgroundIdentifierInvalid) {`

Reply:
Makes sense, thanks.